extensions/oidc-token-propagation-reactive/deployment/src/main/java/io/quarkus/oidc/token/propagation/reactive/OidcTokenPropagationReactiveBuildTimeConfig.java

```diff
@@ -13,4 +13,18 @@ public class OidcTokenPropagationReactiveBuildTimeConfig {
      */
     @ConfigItem(defaultValue = "true")
     public boolean enabled;
+
+    /**
+     * If the OIDC Token Reactive Propagation is enabled during authentication, such as when you need
+     * to use a REST client from {@link io.quarkus.security.identity.SecurityIdentityAugmentor}.
+     * By default, access token propagation is not enabled during authentication, because
+     * current implementation requires storing of local data on Vert.X duplicated context in a safe manner.
+     * It should be safe to enable this option in most cases, but please check that extensions used by your
+     * application do not use root context. The simplest check should be to enable this option and write
+     * integration tests as we raise exception when unsafe Vert.X context is used.
+     * <p>
+     * This configuration options is only supported when used together with the Quarkus OIDC extension.
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean enabledDuringAuth;
```

Comment:
```suggestion
    public boolean enabledDuringAuthentication;
```

Reply:
It is subjective but I thought I'd propose a full form, not significantly more verbose